pkg/webhook/register.go

```diff
@@ -39,7 +39,6 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/apimachinery/pkg/types"
-	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
FYI this allows for authentication using GCP, it isn't safe to remove this in general.  However, in this case yeah shouldn't be needed!